desktop/renderer-app/src/stores/WhiteboardStore.ts

```diff
@@ -152,6 +153,15 @@ export class WhiteboardStore {
                 onDisconnectWithError: error => {
                     console.error(error);
                 },
+                onKickedWithReason: reason => {
+                    if (
+                        reason === "kickByAdmin" ||
+                        reason === "roomDelete" ||
+                        reason === "roomBan"
+                    ) {
+                        this.isKicked = true;
+                    }
```

Comment:
yes.

`replaceByOther` should never occur because there's only one instance of our app.

`roomZombie` and `crash` may occur in bad network, which should not be treat as being kicked out.